lib/config.js

```diff
@@ -39,6 +39,7 @@ function getDefaultConfig() {
   return {
     coverageEnvVar: 'COVERAGE',
     coverageFolder: 'coverage',
+    coveredAddons: [],
```

Comment:
added a config item that allows you to specify which addons.  You can just add it to `coverage.js` in your app config.  
For example:
![image](https://user-images.githubusercontent.com/3609063/67510576-34d3c600-f663-11e9-8c37-95fb36e353df.png)